ReactNativeClient/lib/KeymapUtils.ts

```diff
@@ -0,0 +1,43 @@
+const { shim } = require('lib/shim');
+
+export const DOMtoElectronAccelerator = (e: React.KeyboardEvent<HTMLDivElement>) => {
```

Comment:
`domToElectronAccelerator`. Only classes or components should start with an uppercase.